DQM/L1TMonitorClient/src/L1TEMTFEventInfoClient.cc

```diff
@@ -0,0 +1,1247 @@
+            m_noisyStrip(parSet.getParameter<std::vector<edm::ParameterSet> >("NoisyStrip")),
+            m_deadStrip(parSet.getParameter<std::vector<edm::ParameterSet> >("DeadStrip")),
+            m_disableNoisyStrip(parSet.getParameter<std::vector<std::string> >("DisableNoisyStrip")),
+            m_disableDeadStrip(parSet.getParameter<std::vector<std::string> >("DisableDeadStrip")),
+            m_nrTrackObjects(0),
+            m_nrHitObjects(0),
+            m_nrNoisyStrip(0),
+            m_nrDeadStrip(0),
+            m_totalNrQtSummaryEnabled(0) {
+
+    initialize();
+}
+
+// destructor
+L1TEMTFEventInfoClient::~L1TEMTFEventInfoClient() {
+    //empty
+}
+
+void L1TEMTFEventInfoClient::initialize() {
+  
+    if (m_verbose) {
+        std::cout << "\nMonitor directory =             " << m_monitorDir
+                << std::endl;
+    }
+
+    // L1 systems
+
+    m_nrTrackObjects = m_trackObjects.size();
+
+    m_trackLabel.reserve(m_nrTrackObjects);
+    // m_trackLabelExt.reserve(m_nrTrackObjects);  // Not needed? - AWB 05.12.16
+    m_trackDisable.reserve(m_nrTrackObjects);
+
+    // on average five quality test per system - just a best guess
+    m_trackQualityTestName.reserve(5*m_nrTrackObjects);  // Not needed? - AWB 05.12.16
+    m_trackQualityTestHist.reserve(5*m_nrTrackObjects);  // Not needed? - AWB 05.12.16
+    m_trackQtSummaryEnabled.reserve(5*m_nrTrackObjects);  // Not needed? - AWB 05.12.16
+
+    int indexSys = 0;
+
+    int totalNrQualityTests = 0;
+
+    for (std::vector<edm::ParameterSet>::const_iterator itTrack =
+            m_trackObjects.begin(); itTrack != m_trackObjects.end(); ++itTrack) {
+
+        m_trackLabel.push_back(itTrack->getParameter<std::string>(
+                "SystemLabel"));
+
+        // m_trackLabelExt.push_back(itTrack->getParameter<std::string>(  // Not needed? - AWB 05.12.16
+        //         "HwValLabel"));
+
+        m_trackDisable.push_back(itTrack->getParameter<unsigned int>(
+                "SystemDisable"));
+        // check the additional disable flag from m_disableTrackObjects
+        for (std::vector<std::string>::const_iterator itSys =
+                m_disableTrackObjects.begin(); itSys != m_disableTrackObjects.end(); ++itSys) {
+
+            if (*itSys == m_trackLabel[indexSys]) {
+                m_trackDisable[indexSys] = 1;
+
+            }
+        }
+
+        std::vector < edm::ParameterSet > qTests = itTrack->getParameter<
+                std::vector<edm::ParameterSet> > ("QualityTests");
+        size_t qtPerSystem = qTests.size();
+
+        std::vector < std::string > qtNames;
+        qtNames.reserve(qtPerSystem);
+
+        std::vector < std::string > qtFullPathHists;
+        qtFullPathHists.reserve(qtPerSystem);
+
+        std::vector<unsigned int> qtSumEnabled;
+        qtSumEnabled.reserve(qtPerSystem);
+
+	std::cout << "\nLooping over track quality tests" << std::endl;
```

Comment:
Remove all std::couts if they are not behind an `if (verbose)` .

Reply:
Done